extra/2-piping.js

```diff
@@ -16,27 +16,25 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  return num1 + num2;
 }
 
-function multiply() {
-
+function multiply(num1, num2) {
+  return num1 * num2;
 }
 
-function format() {
-
+function format(num) {
+  return "£" + num;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
-
+let badCode = format(multiply(add(startingValue, 10), 2));
```

Comment:
I believe the idea here is to do the multiply and add on separate lines using variables to store the values. This makes the code easier to read and to debug.